### ISSUE TITLE
accessLogsFile and traefikLogsFile not respected when set via cli

### ISSUE THREAD

**Issue by egalano** [containous/traefik#405]
When running version v1.0.0-beta.756 setting the accessLogsFile and traefikLogsFile arguments has no effect. Setting it via the config file worked properly.

```
root@localhost ~ # ./traefik --logLevel DEBUG --accessLogsFile foo.log --traefikLogsFile bar.log --consulCatalog.endpoint ${CONSUL_SERVER}
2016/05/25 00:33:59 configuration.go:216: No configuration file found
DEBU[2016-05-25T00:33:59Z] Global configuration loaded {"GraceTimeOut":10,"AccessLogsFile":"","TraefikLogsFile":"","LogLevel":"DEBUG","EntryPoints":{"http":{"Network":"","Address":":80","TLS":null,"Redirect":null}},"ACME":null,"DefaultEntryPoints":["http"],"ProvidersThrottleDuration":2000000000,"MaxIdleConnsPerHost":200,"Retry":null,"Docker":null,"File":null,"Web":null,"Marathon":null,"Consul":null,"ConsulCatalog":null,"Etcd":null,"Zookeeper":null,"Boltdb":null,"Kubernetes":null}
INFO[2016-05-25T00:33:59Z] Preparing server http &{Network: Address::80 TLS:<nil> Redirect:<nil>}
INFO[2016-05-25T00:33:59Z] Starting server on :80
```

**Comment by emilevauge** [comment]
@egalano could you try with v1.0.0-rc2?

**Comment by egalano** [comment]
Thanks. Confirmed fixed in 1.0.0-rc2
